pkg/operator/hive/hive_controller.go

```diff
@@ -110,6 +110,11 @@ func mapToHiveConfig[T client.Object](r reconcile.Reconciler, src string) func(c
 	}
 }
 
+// TODO: This may or may not be setting fieldManager.
```

Comment:
I prefer the `todo` line to have at least an action item, so anyone who wants to work on it in the future has a clear sense of what to implement in order to remove this line.

Reply:
I looked deeper and am _pretty_ sure none of our paths will fall back to using user-agent, so I reverted this and its callers.